test/e2e/coverage_docs/E2E_coverage.md

```diff
@@ -0,0 +1,51 @@
+### E2E coverage of features
```

Comment:
I think we can do it after each of our file analyses is completed, then we can analyze the overall characteristics of E2E test coverage.

Reply:
Yes, then I will temporarily keep this file, but delete the current content